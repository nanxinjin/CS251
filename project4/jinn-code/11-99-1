1
1
1
1
2
2
3
4
5
6
8
9
A
C
C
C
D
F
G
G
G
G
H
H
H
I
I
I
J
K
K
L
L
M
N
O
O
O
P
R
S
U
U
V
W
W
W
W
W
X
Z
a
a
b
b
b
b
c
d
d
d
e
e
g
g
h
j
j
j
l
l
m
n
n
n
n
n
n
p
p
q
q
r
r
s
t
t
u
u
u
v
v
w
x
x
x
x
x
y
