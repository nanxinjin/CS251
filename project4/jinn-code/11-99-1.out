missing
found
found
missing
found
found
missing
missing
found
found
found
