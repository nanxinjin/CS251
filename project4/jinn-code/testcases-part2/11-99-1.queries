_
A
H
_
R
R
c
_
g
j
t
