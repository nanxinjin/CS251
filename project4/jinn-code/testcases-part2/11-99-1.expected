missing
found
found
missing
found
found
found
missing
found
found
found
